macros/edr/dbt_artifacts/dbt_columns/upload_dbt_columns.sql

```diff
@@ -0,0 +1,11 @@
+{% macro upload_dbt_columns() %}
+  {% set relation = elementary.get_elementary_relation('dbt_columns') %}
+  {% if execute and relation %}
+    {% set query %}
```

Comment:
Use [`create_or_replace`](https://github.com/elementary-data/dbt-data-reliability/blob/3c4ce9e5d646ae9ac48fbf247a439d192c645038/macros/utils/table_operations/create_or_replace.sql#L0-L1).

Reply:
What do you mean? There's a link in my comment, lol.